assets/tests/components/map/controls/LayersControl.test.tsx

```diff
@@ -0,0 +1,99 @@
+import { render, screen } from "@testing-library/react"
+import "@testing-library/jest-dom"
+import React, { ReactNode } from "react"
+import { MapContainer } from "react-leaflet"
+import { LayersControl } from "../../../../src/components/map/controls/LayersControl"
+import userEvent from "@testing-library/user-event"
+import { mockTileUrls } from "../../../testHelpers/mockHelpers"
+import { layersControlButton } from "../../../testHelpers/selectors/components/map"
+
+jest.mock("../../../../src/tilesetUrls", () => ({
+  __esModule: true,
+  tilesetUrlForType: jest.fn(() => null),
+}))
+
+const mapWrapper = ({ children }: { children: ReactNode }) => (
+  <MapContainer center={[0, 0]} zoom={0}>
+    {children}
+  </MapContainer>
+)
+
+beforeAll(() => {
+  mockTileUrls()
+})
+
+describe("LayersControl", () => {
+  test("when the control button is clicked, the list of layers is rendered with the selected layer marked as selected", async () => {
+    render(<LayersControl tileType="base" setTileType={jest.fn} />, {
+      wrapper: mapWrapper,
+    })
+    await userEvent.click(layersControlButton.get())
+
+    expect(screen.getByLabelText("Map (default)")).toBeChecked()
+    expect(screen.getByLabelText("Satellite")).not.toBeChecked()
+  })
+
+  test("when the a tile option is clicked, then setTileType is called", async () => {
+    const setTileTypeMock = jest.fn()
+    render(<LayersControl tileType="base" setTileType={setTileTypeMock} />, {
+      wrapper: mapWrapper,
+    })
+    await userEvent.click(layersControlButton.get())
+    await userEvent.click(screen.getByLabelText("Satellite"))
+
+    expect(setTileTypeMock).toHaveBeenCalled()
```

Comment:
**nitpick (non-blocking):** you could use `toHaveBeenCalledWith` to also assert that it's called with the correct value.